packages/core/src/parser/parser.ts

```diff
@@ -246,8 +282,8 @@ export class Parser extends CstParser {
         this.SUBRULE1(this.callExpression);
         this.MANY({
             DEF: () => {
-                this.CONSUME(Dot);
-                this.SUBRULE2(this.simpleCallExpression);
+                this.CONSUME(Dot, { ERR_MSG: "Expected a '.' but found none" });
+                this.withError(() => this.SUBRULE2(this.simpleCallExpression), `Identifiers may not end with a '.'`);
```

Comment:
tbh I would rather have expected sth like `missing identifier after '.'` or so, but if you prefer it like this, fine for me